package.json

```diff
@@ -5,6 +5,8 @@
     "@emotion/react": "11.11.1",
     "@lexical/history": "0.12.2",
     "@lexical/react": "0.12.2",
+    "@loadable/component": "^5.15.3",
```

Comment:
can you also explain why this is needed now?

Reply:
gets removed